app/modules/users/scss/contribution.scss

```diff
@@ -62,10 +72,25 @@
       &:before{ content: '- ' }
     }
   }
+  .user{
+    @include display-flex(row, center, flex-start);
+    @include transition(background-color);
+    margin-left: 0.5em;
+    @include radius;
+    &:hover{
+      background-color: $off-white;
+    }
+    padding: 0.2em;
+    img{
+      @include radius;
+      margin-right: 0.5em;
+    }
+  }
   .time{
     color: $grey;
-    margin-left: 0.5em;
-    width: 7em;
+    margin-left: auto;
+    padding-left: 0.2em;
+    width: 6em;
     text-align: right;
   }
   .togglers span{
```

Comment:
nice!
how about modifying `.operation-type` right below to have some more accessible label

from

![2020-11-20_16-50](https://user-images.githubusercontent.com/5363918/99820068-2eef3380-2b48-11eb-960f-8f67fd36cb0b.png)

to something like

![2020-11-20_16-51](https://user-images.githubusercontent.com/5363918/99820195-56460080-2b48-11eb-92e0-59a62687f38a.png)

Reply:
better? 1c585b5c

![Sélection_053](https://user-images.githubusercontent.com/1596934/99876746-fbb8ad00-2bf8-11eb-9419-035293cb12b3.jpg)

Reply:
or with a white background
![Sélection_054](https://user-images.githubusercontent.com/1596934/99876771-3589b380-2bf9-11eb-906c-1c5619c78838.jpg)

Reply:
nice, either way